### ISSUE TITLE
Connected Devices not populated properly even though switching still works

### ISSUE THREAD

**Issue by jdhorner** [sirWest/AudioSwitch#131]
Recently, an issue where the devices aren't listed properly started happening when opening the full list. Quick-switching still works, but they lists of playback and recording devices aren't populated properly. (Some are missing, and none have maintained my previously-configured display names and colors)

It's easier to show with a screenshot. (attached)

Running Windows 10, with a 4k Dell P2715Q display powered by an AMD R9 280X graphics card. 

<img width="405" alt="2017-08-29 22_45_37-audioswitch - settings" src="https://user-images.githubusercontent.com/2692640/29853199-79de8cac-8d0c-11e7-82ea-9a9f1d222707.png">
<img width="405" alt="2017-08-29 22_45_56-audioswitch - settings" src="https://user-images.githubusercontent.com/2692640/29853200-79df2d60-8d0c-11e7-9d30-6f608d183aa6.png">

**Comment by nogweii** [comment]
Yeah, I'm seeing this as well. Windows 10 (version 1703, Build 15063.540), BenQ PD3200U 32" 4K monitor, Nvidia Geforce GTX 970. I have Windows set to 150% scaling in my display settings.

This is probably a duplicate of #109.
